sunpy/net/attr.py

```diff
@@ -53,6 +56,46 @@ def strtonum(value):
             'seven', 'eight', 'nine')[value]
 
 
+def _print_attrs(attr):
```

Comment:
I think it would be good to also facilitate `_repr_html_` for notebook output. To be able to do this, I think you will need a function to output a table, and then deal with the prepending thing for str and `html_repr` separately.

Reply:
Done.